src/SFA.DAS.AANHub.Application/Common/Commands/BaseMemberCommand.cs

```diff
@@ -5,13 +5,14 @@ namespace SFA.DAS.AANHub.Application.Common.Commands
 {
     public abstract class BaseMemberCommand : IBaseMemberCommand
     {
-        public string? Id { get; set; }
-        public MembershipUserTypes? UserType { get; set; }
+        protected BaseMemberCommand() => Id = Guid.NewGuid();
+        public Guid Id { get; }
+        public MembershipUserType? UserType { get; set; }
```

Comment:
I've left this in for now as the old `CreateMember` code still uses it. It can be removed when deprecated. I've hard coded UserType in the command though

Reply:
Can it be moved to CreateMemberCommand then?

Reply:
That way it will go away automatically when that command is removed.

Reply:
Of course! Moved